services/transition/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-transition: &transition
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: transition
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/transition
+
+services:
+  transition-lite:
+    <<: *transition
+    depends_on:
+      - nginx-proxy-app
+      - postgres
+      - redis
+      - transition-worker
```

Comment:
We shouldn't need this (or `nginx-proxy-app`) here. The dependencies for the `lite` stack should be only those necessary to get the tests passing.

https://github.com/alphagov/govuk-docker#stacks